public/content/eips/index.md

```diff
@@ -54,10 +54,20 @@ Full standardization process alongside with chart is described in [EIP-1](https:
 
 If you’re interested to read more about EIPs, check out the [EIPs website](https://eips.ethereum.org/) and [EIP-1](https://eips.ethereum.org/EIPS/eip-1). Here are some useful links:
 
+### Official EIP resources
+
 - [A list of every EIP](https://eips.ethereum.org/all)
 - [A description of all EIP types](https://eips.ethereum.org/EIPS/eip-1#eip-types)
 - [A description of all EIP statuses](https://eips.ethereum.org/EIPS/eip-1#eip-process)
```

Comment:
The same comment as above. It doesn't seem necessary to me.

Reply:
I added PEEPanEIP to the community education section. Now that the EIPs and ERCs repositories are apart, I can add a link pointing to ercs.ethereum.org and remove the duplicate links from the existing official EIP resources section. Thoughts?